src/lsst/rsp/utils.py

```diff
@@ -84,7 +131,7 @@ def jupyter_proxy_url(port: Optional[int] = None) -> str:
     # The type: ignore is needed because the alternate form of notebook_url
     # (https://docs.bokeh.org/en/latest/docs/reference/io.html#bokeh.io.show)
     # isn't in bokeh's type hints.
-    bokeh.io.show(obj=obj, notebook_url=jupyter_proxy_url)  # type:ignore
+    bokeh.io.show(obj=obj, notebook_url=jupyter_proxy_url)
```

Comment:
Did they finally fix this?  If they did, kill off the comment before it too.

Reply:
Yes, the linter said that it was unused, I'll kill the comment